src/convolution_api.cpp

```diff
@@ -343,6 +344,15 @@ static void LogCmdConvolution(const miopenTensorDescriptor_t xDesc,
                 << " -v " << miopen::deref(convDesc).GetConvStrides()[1]
                 << " -l " << miopen::deref(convDesc).GetConvDilations()[0]
                 << " -j " << miopen::deref(convDesc).GetConvDilations()[1]; // clang-format on
+            std::string x_layout = miopen::deref(xDesc).GetLayout("NCHW");
+            std::string w_layout = miopen::deref(wDesc).GetLayout("NCHW");
+            std::string y_layout = miopen::deref(yDesc).GetLayout("NCHW");
+            if(x_layout != "NCHW")
```

Comment:
Lets avoid the check and print the layout regardless ?

Reply:
If default layouts are not printed, than it will be easier for human readers

Reply:
Yes... by default layout is not printout, so here I just not print out if current is indeed `NCHW`